docs/Makefile

```diff
@@ -12,7 +12,11 @@ BUILDDIR      = build
 help:
 	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
-.PHONY: help Makefile
+clean:
+	@echo "Removing everything under 'build' and 'source/generated'..."
+	@rm -rf build/ source/generated
```

Comment:
now I want to go add this line to the other repos... they typically only clean up the build folder for some reason